### ISSUE TITLE
Data normalization

### ISSUE THREAD

**Issue by aaronhsueh0506** [Rikorose/DeepFilterNet#58]
Hello,
I am trying to do my own data augmentation.
And I found that my dataset always has Nan loss 3 loss function.
I noticed that the paper says that the data has been exponentially mean/unit normalized, I think maybe this cause the Nan issue for me. Could you give some details?

Thanks,

**Comment by Rikorose** [comment]
Hi the normalization uses this code:
https://github.com/Rikorose/DeepFilterNet/blob/a182ec7/libDF/src/lib.rs#L174-L189

I don't think thats an issue though. Have you tried running with torch's anomaly detection? What error stack do you get?

**Comment by aaronhsueh0506** [comment]
Ok. On the other hands, the factor of MaskLoss is 0. It this means the MaskLoss didn't work?
Thanks,

**Comment by Rikorose** [comment]
It means, the MaskLoss is not computed and does not result in any loss/gradient.

**Comment by aaronhsueh0506** [comment]
Does ERB and spectrogram normalization seriously affect the results?

Now I train the model on Keras, but the result is not good. PESQ is worse than unprocessed.

Thanks,
